### PR TITLE
 #164798213 Enable users to add tags to their articles

### PR DESCRIPTION
**Pull request by ezrogha**
#### What does this PR do?
Add tags to Articles

#### Description of Task to be completed?
Enable users add tags to their articles

#### How should this be manually tested?
Fetch branch `git fetch origin ft-add-tags-to-articles-164798213`
Checkout `git checkout ft-add-tags-to-articles-164798213`
Run `python manage.py makemigrations`
Run `python manage.py migrate`
Run `pipenv shell` to install packages
Run `python manage.py runserver`

#### What are the relevant pivotal tracker stories?
[#164798213](https://www.pivotaltracker.com/story/show/164798213)

#### Screenshots:
Screenshot 2019-04-09 at 18 28 48

#### Postman Collection
[Run in Postman](https://app.getpostman.com/run-collection/362baeb920402e3d3f54)

### CODE REVIEW

File: authors/apps/articles/migrations/0002_article_tags.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2 on 2019-04-09 15:04
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
```
Comment (dannylwe) [inline authors/apps/articles/migrations/0002_article_tags.py]: Hi @ezrogha 
I see you included migration files some of which already exist on develop
kindly delete them and leave only the new ones(those not on develop)

**Review by dannylwe — CHANGES_REQUESTED** [review]

File: authors/apps/articles/migrations/0002_article_tags.py (hunk 1 above)
Comment (ezrogha) [reply authors/apps/articles/migrations/0002_article_tags.py]: Thanks @dannylwe, 
I have worked on it

**Comment by ja-odur** [comment]
@ezrogha your PR title does not follow convention

**Comment by ezrogha** [comment]
> @ezrogha your PR title does not follow convention

Thanks for the observation @ja-odur,
I have worked on it

**Review by dannylwe — APPROVED** [review]

**Review by ja-odur — APPROVED** [review]
LGTM

**Review by dorothyas — APPROVED** [review]

**Pull request merged**